docs/best_practices.md

```diff
@@ -0,0 +1,130 @@
+<!--[metadata]>
++++
+title = "Best Practices for Using Notary"
+description = "A set of recommended practices for key management, delegating trust, bootstrapping repos, and more."
+keywords = ["docker, Notary, notary-client, docker content trust, content trust", "best practices", "recommended use"]
+[menu.main]
+parent="mn_notary"
+weight=1
++++
+<![end-metadata]-->
+
+# Best Practices for Using Docker Notary
+
+This document describes good and recommended practices for using the Notary client.
+It includes recommendations on key management, bootstrapping trust for your repositories,
+delegating trust to other signers. You may wish to refer to the [Getting Started](getting_started.md)
+and [Advanced Usage](advanced_usage.md) documents for more information on some of the
+commands.
+
+## Key Management
+
+There are five primary key roles in Notary. In order of how critical they are to protect, most
+to least, they are:
+
+1. Root
+2. Targets
+3. Delegations
+4. Snapshot
+5. Timestamp
+
+We will address each of these classifications of key one by one in reverse order. The
+Targets and Delegations key will be covered together as they have almost identical
+security profiles and use cases.
+
+### Timestamp Key
+
+It is required that the server manages your Timestamp key. This is a convenience measure
+as it would be impractical to have a human involved in regularly signing the Timestamp.
+
+### Snapshot Key
+
+You may choose to allow the server to manage your Snapshot key. This behaviour is default
+in the Docker Content trust integration. Typically if you are working on your own, and
+publish new content regularly, it is safe for you to manage the Snapshot key yourself.
+If however you use Delegations to collaborate with other contributors, you will want to
+allow the server to sign snapshots.
+
+We consider holding the Timestamp and Snapshot keys in well secured online locations an
```

Comment:
I have no idea and the online dictionaries aren't helping :-/

Definition has "well-secured" but the examples of use all have "well secured"